cmd/testrunner.go

```diff
@@ -127,6 +128,11 @@ func testTypeCommandActionFactory(runner testrunner.TestRunner) cobraext.Command
 			return cobraext.FlagParsingError(err, cobraext.TestCoverageFlagName)
 		}
 
+		testCoverageType, err := cmd.Flags().GetString(cobraext.TestCoverageFormatFlagName)
```

Comment:
At some point we should check if the given value is one of the allowed ones.

Reply:
Added check about this 

Example of the error
```shell
 $ elastic-package test asset -v --test-coverage --coverage-format=gen
2024/01/18 11:18:11 DEBUG Enable verbose logging
2024/01/18 11:18:11 DEBUG Distribution built without a version tag, can't determine release chronology. Please consider using official releases at https://github.com/elastic/elastic-package/releases
Run asset tests for the package
Error: error parsing --coverage-format flag: coverage format not available: gen

```